stdlib/LinearAlgebra/src/svd.jl

```diff
@@ -26,10 +26,10 @@ julia> F = svd(A)
 SVD{Float64, Float64, Matrix{Float64}, Vector{Float64}}
 U factor:
 4×4 Matrix{Float64}:
- 0.0  1.0  0.0   0.0
```

Comment:
@ViralBShah this doctest was failing and I had to update it.  Can you please double check this change is acceptable?

Reply:
Yes it is fine.